oauth2client/contrib/django_util/__init__.py

```diff
@@ -212,6 +286,33 @@ def _get_oauth2_client_id_and_secret(settings_instance):
                 "GOOGLE_OAUTH2_CLIENT_SECRET in settings.py")
 
 
+def _get_storage_model():
+    """This configures whether the credentials will be stored in the session
+    or the Django ORM based on the settings. By default, the credentials
+    will be stored in the session, unless `GOOGLE_OAUTH2_STORAGE_MODEL`
+    is found in the settings. Usually, the ORM storage is used to integrate
+    credentials into an existing Django user system.
+
+    Returns:
+       A tuple containing three strings, or None. If
```

Comment:
Weird three-space indent on this line followed by single-space indent on additional lines. Dedent this one line by one space and this will be good.

(And one day an autoformatter will do this for us.)